build.gradle

```diff
@@ -0,0 +1,171 @@
+buildscript {
+    ext {
+        TESTNG_VERSION = '7.3.0'
+        CUCUMBER_VERSION = '6.7.0'
+       	APPLICATION_URL = 'http://bstackdemo.com/'
+       	APPLICATION_LOCAL_URL = 'http://localhost:3000/'
+        ALLURE_VERSION = "2.14.0"
+        ALLURE_GRADLE_VERSION = "2.8.1"
+        SELENIUM_VERSION = "3.141.59"
+        BROWSERSTACK_LOCAL_VERSION="1.0.6"
+        APACHE_COMMON_VERSION = "3.11"
+        SL4J_VERSION = "1.7.30"
+        LOGBACK_VERSION = "1.2.3"
+        TESTNG_VERSION="7.4.0"
+        CUCUMBER_VERSION = "6.10.4"
+        JACKSON_DATABIND_VERSION = "2.12.2"
+      	THREAD_COUNT = "8"
+       	
+    }
+}
+
+
+plugins {
+    id 'java'
+    id "io.qameta.allure" version "2.8.1"
+
+}
+
+
+version '1.0-SNAPSHOT'
+repositories {
+    mavenCentral()
+    maven {
+        url "https://plugins.gradle.org/m2/"
+    }
+
+}
+dependencies {
+    testImplementation group:'org.seleniumhq.selenium', name:'selenium-java', version:"$SELENIUM_VERSION"
```

Comment:
Commons Lang3 missing from dependencies.
Gradle profiles not working.

Reply:
Fixed this